app/build.gradle

```diff
@@ -39,6 +39,7 @@ dependencies {
     implementation 'androidx.appcompat:appcompat:1.2.0'
     implementation 'com.google.android.material:material:1.2.1'
     implementation 'androidx.constraintlayout:constraintlayout:2.0.4'
+    implementation 'androidx.legacy:legacy-support-v4:1.0.0'
```

Comment:
why need this dependency ?

Reply:
It was added by android studio. I did not noticed on which step. I removed it in next commit - nothing was broken.

Reply:
okay good